### PR TITLE
Aizad/73977/binary link ts

### PR DESCRIPTION
**Pull request by aizad-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/reports/src/Components/Routes/__tests__/helpers.spec.tsx [hunk 1]
```diff
@@ -1,31 +1,34 @@
-import React from 'react';
 import * as Helpers from '../helpers';
 import { routes } from '@deriv/shared';
-import getRoutesConfig from 'Constants/routes-config';
+import getRouteConfig from 'Constants/routes-config';
```
Comment (akmal-deriv) [inline packages/reports/src/Components/Routes/__tests__/helpers.spec.tsx]: Did you mistakenly rename this function?

Comment (aizad-deriv) [reply packages/reports/src/Components/Routes/__tests__/helpers.spec.tsx]: Resolved 👍🏼

File: packages/reports/src/Components/Routes/__tests__/helpers.spec.tsx [hunk 2]
```diff
@@ -1,31 +1,34 @@
-import React from 'react';
 import * as Helpers from '../helpers';
 import { routes } from '@deriv/shared';
 import getRoutesConfig from 'Constants/routes-config';
 
+const routeConfig = getRoutesConfig({ is_appstore: false, is_pre_appstore: false });
```
Comment (mahdiyeh-deriv) [inline packages/reports/src/Components/Routes/__tests__/helpers.spec.tsx]: why you are passing is_appstore and is_pre_ppstore?

Comment (mahdiyeh-deriv) [reply packages/reports/src/Components/Routes/__tests__/helpers.spec.tsx]: also I suggest to rename it to `routesConfig`

**Review by mahdiyeh-deriv — APPROVED** [review]

**Pull request merged**